docs/source/en/api/pipelines/cogvideox.md

```diff
@@ -33,9 +33,7 @@ This pipeline was contributed by [zRzRzRzRzRzRzR](https://github.com/zRzRzRzRzRz
 
 ## Inference
 
-Use [`torch.compile`](https://huggingface.co/docs/diffusers/main/en/tutorials/fast_diffusion#torchcompile) to reduce the inference latency.
-
-First, load the pipeline:
+Inference can be run with CogVideoX by following the code below:
```

Comment:
We need to separately embed an inference code snippet no? This is because we already have it available from the pipeline call. The API pipeline spec should be kept as clean as possible.

Reply:
Do you mean to remove the following lines here?

![image](https://github.com/user-attachments/assets/1dd49e27-e7f6-4c16-bb23-edbf4174e7ac)